packages/expo-location/src/Location.ts

```diff
@@ -159,8 +171,39 @@ export async function enableNetworkProviderAsync(): Promise<void> {
 }
 
 export async function getCurrentPositionAsync(
-  options: LocationOptions = {}
+  options: LocationOptions = {},
+  methodOptions: LocationMethod = { type: LocationMethodType.Fast }
 ): Promise<LocationData> {
+  console.log(methodOptions);
+  if (methodOptions.type === LocationMethodType.Fast) {
+    console.log("fast");
+    return new Promise<LocationData>(async (resolve, reject) => {
+      try {
+        let bestPosition: LocationData | null = null;
```

Comment:
IMO it's strange initialisation, I would rather go with
```suggestion
        let bestPosition: LocationData | undefined;
```